src/app/share/views/ShareView/ShareFolderView.tsx

```diff
@@ -121,8 +119,10 @@ export default function ShareFolderView(props: ShareViewProps): JSX.Element {
       .then((folderSize) => {
         setSize(folderSize);
       })
-      .catch((err) => {
+      .catch(async (err) => {
         if (err.message === 'Forbidden') {
+          const itemData = await shareService.getPublicSharedItemInfo(sharingId);
+          setItemData(itemData);
```

Comment:
Same here, maybe it is better to extract this to a function to control possible errors.